utils/src/tree/isAzExtParentTreeItem.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { AzExtParentTreeItem } from "./AzExtParentTreeItem";
+import { AzExtTreeItem } from "./AzExtTreeItem";
+import type { IAzExtParentTreeItemInternal } from "./InternalInterfaces";
+
+/**
+ * Using instanceof AzExtParentTreeItem causes issues since each extension has their own version of the utils. Instead, check _isAzExtParentTreeItem
+ */
+export function isAzExtParentTreeItem(maybeParentTreeItem: unknown): maybeParentTreeItem is AzExtParentTreeItem {
```

Comment:
If we end up with any more of these I'd suggest switching to something like `_azExtParentTreeItemType` with an enum/flags type.

Reply:
I don't think we're planning on adding more of these.